### PR TITLE
Cross-compile + update Readme with release steps + CI

### PR DESCRIPTION
**Pull request by rreas**
Moves to cross-compilation similar to other PRs 

* https://github.com/allenai/common/pull/225
* https://github.com/allenai/datastore/pull/34
* https://github.com/allenai/Word2VecJava/pull/5

### CODE REVIEW

**Comment by rreas** [comment]
Tests passing on CI; only formatting fails so leaving for a follow-up

File: project/build.properties [hunk 1]
```diff
@@ -1 +1 @@
-sbt.version=0.13.8
\ No newline at end of file
+sbt.version=1.2.8
```
Comment (bbstilson) [inline project/build.properties]: 💯

File: project/plugins.sbt [hunk 2]
```diff
@@ -1,5 +1,5 @@
-addSbtPlugin("org.allenai.plugins" % "allenai-sbt-plugins" % "1.5.0")
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.10")
 
-addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.3")
+addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.0.4")
 
-addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.1")
+addSbtPlugin("org.foundweekends" % "sbt-bintray" % "0.5.4")
```
Comment (bbstilson) [inline project/plugins.sbt]: How come these changed?

Comment (rreas) [reply project/plugins.sbt]: PGP used for the sbt-release process. The Bintray release process doesn't seem to sign and instead relies on our local Bintray credentials. I'm not sure beyond that (admittedly this is a bit of a disappointing stopping point). However, it works, and I didn't want to spend too much time on this as the recommendation seems to be to use bintray for releases.

**Pull request merged**